tests/test_cyto_utils/conftest.py

```diff
@@ -113,27 +113,3 @@ def fixture_cell_loc_obj3(
         metadata_input=metadata_input_file_s3,
         single_cell_input=single_cell_input_file_s3,
     )
-
-
-@pytest.fixture(name="cell_loc1")
-def fixture_cell_loc1(cell_loc_obj1: CellLocation) -> pd.DataFrame:
-    """
-    Provide the output of running CellLocation.add_cell_location
-    """
-    return cell_loc_obj1.add_cell_location()
-
-
-@pytest.fixture(name="cell_loc2")
-def fixture_cell_loc2(cell_loc_obj2: CellLocation) -> pd.DataFrame:
-    """
-    Provide the output of running CellLocation.add_cell_location
-    """
-    return cell_loc_obj2.add_cell_location()
-
-
-@pytest.fixture(name="cell_loc3")
-def fixture_cell_loc3(cell_loc_obj3: CellLocation) -> pd.DataFrame:
-    """
-    Provide the output of running CellLocation.add_cell_location
-    """
-    return cell_loc_obj3.add_cell_location()
```

Comment:
Big fan of this change. These fixtures were pretty confusing to figure out when I was doing the linting changes.

Reply:
I had no idea what I was doing here :D